cfgov/jinja2/v1/index.html

```diff
@@ -181,6 +181,7 @@ <h3 class="h1 m-card_heading">97% of consumers</h3>
             <div class="content_main content_main__home">
                 <h2>Latest updates</h2>
                 <div class="content-l">
+                    {% import 'macros/category-icon.html' as category_icon %}
```

Comment:
Why is this `as category_icon` here? `category_icon` is not used in this file.

Reply:
That's a good question, I have no idea, but I'll remove it.